packages/jest-each/src/bind.js

```diff
@@ -195,6 +197,11 @@ const getMatchingKeyPaths = title => (matches, key) =>
 const replaceKeyPathWithValue = data => (title, match) => {
   const keyPath = match.replace('$', '').split('.');
   const value = getPath(data, keyPath);
+  const valueType = getType(value);
+
+  if (PRIMITIVES.indexOf(valueType) > -1) {
```

Comment:
`includes`?

Reply:
Couldn't remember if Node 6 supported it but looks like it does so I'll update! 👍